models/auth/oauth2.go

```diff
@@ -512,9 +512,14 @@ func GetActiveOAuth2ProviderSources() ([]*Source, error) {
 func GetActiveOAuth2SourceByName(name string) (*Source, error) {
 	authSource := new(Source)
 	has, err := db.GetEngine(db.DefaultContext).Where("name = ? and type = ? and is_active = ?", name, OAuth2, true).Get(authSource)
-	if !has || err != nil {
+	
+	if err != nil {
 		return nil, err
-	}
+  }
+  
+	if !has {
+		return nil, fmt.Errorf("Not found")
```

Comment:
The error message should have more detail.

Reply:
Added detail